dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTracker.h

```diff
@@ -39,23 +40,12 @@ namespace winrt::Microsoft::Windows::System::implementation
 
         wil::unique_hkey GetKeyForTrackingChange(DWORD* disposition) const
         {
-            HKEY topLevelKey{};
-
-            if (m_Scope == EnvironmentManager::Scope::User)
-            {
-                topLevelKey = HKEY_CURRENT_USER;
-            }
-            else
-            {
-                topLevelKey = HKEY_LOCAL_MACHINE;
-            }
-
-            auto subKey{ wil::str_printf<wil::unique_cotaskmem_string>(
-                L"Software\\ChangeTracker\\%ws\\%ws\\%ws\\", KeyName(), m_PackageFullName.c_str(), m_Key.c_str()) };
+            std::filesystem::path subKey = std::filesystem::path{L"Software\\ChangeTracker"}
+            / KeyName() / m_PackageFullName / ScopeToString() / m_Key;
```

Comment:
NIT: Indent 4 spaces

```
auto f{ std::filesystem::path{L"blahblah"}
    / foo
    / bar;
```

As-is it looks like 2 statements

same comment applies throughout